### ISSUE TITLE
Potentially unnecessary code in `svelte.config.js`

### ISSUE THREAD

**Issue by benmccann** [sveltejs/eslint-plugin-svelte#1294]
https://github.com/sveltejs/eslint-plugin-svelte/blob/c2a8820029565c28fde37e9c9a786f8d50cc01ea/docs-svelte-kit/svelte.config.js#L9 does the following:

```js
const dirname = path.dirname(fileURLToPath(import.meta.url));
...
		files: {
			routes: path.join(dirname, './src/routes'),
			appTemplate: path.join(dirname, './src/app.html'),
			hooks: {
				server: path.join(dirname, './src/hooks/server'),
				client: path.join(dirname, './src/hooks/client')
			},
			lib: path.join(dirname, './src/lib'),
			assets: path.join(dirname, './statics')
		}
```

This code seems like it should all be able to be deleted as it appears to be doing the default behavior. Maybe the `cwd` is set incorrectly when running? Can the `dev` and `build` commands be run from the `docs-svelte-kit` directory?

I'm asking because we hope to deprecate these options

**Comment by ota-meshi** [comment]
Ah, those are no longer needed 😅

**Comment by baseballyama** [comment]
close in https://github.com/sveltejs/eslint-plugin-svelte/pull/1309
